### ISSUE TITLE
Coordinates of STORM Experiments

### ISSUE THREAD

**Issue by CHB-Bohrer** [BogdanBintu/ChromatinImaging#2]
Maybe I am wrong, but it seems as though there are only xyz cords from the diffraction-limited experiments?

**Comment by manuelfmerino** [comment]
> 
> 
> Maybe I am wrong, but it seems as though there are only xyz cords from the diffraction-limited experiments?

Agree! Would be great to actually get the xyz cords of the STORM experiments on IMR90 cells.
